src/datasets/non_desktop/apple_app_store/reference.md

```diff
@@ -0,0 +1,115 @@
+# Apple App Store
+<!-- toc -->
+
+# Introduction
+The [`apple_app_store`](https://console.cloud.google.com/bigquery?project=moz-fx-data-marketing-prod&folder=&organizationId=&p=moz-fx-data-marketing-prod&d=apple_app_store&page=dataset) dataset is used to understand the acquisition performance for the nondesktop products through the Apple App Store along the key metrics and dimensions. Apple’s documentation for all metrics and dimensions can be found [here](https://help.apple.com/app-store-connect/#/itc21781223f).
+
+# Contents
+The [`apple_app_store`](https://console.cloud.google.com/bigquery?project=moz-fx-data-marketing-prod&folder=&organizationId=&p=moz-fx-data-marketing-prod&d=apple_app_store&page=dataset) dataset contains a collection of aggregated tables by a singular dimension that explains the performance of acquisition activity through the Apple App Store. 
+
+**IMPORTANT NOTE:** The data is received from Apple with only one dimension per metric. As a result, we are unable to do multi-dimensional analysis. i.e. we can tell how each storefront is performing but we can’t further breakdown the storefront to see how specific platforms or sources are contributing to that storefront’s performance. 
+
```

Comment:
The same caveat is repeated below under background and caveats, let's skip it here

```suggestion
```